### PR TITLE
js files that needed asset function prepend

### PR DESCRIPTION
**Pull request by Cupidazul**
svg-dynamic-graphs url fix using asset internal function 

and other snipets are missing the asset function prepend.

This need is to add : ' hxxp(s):/xxxx   '  to all script 'src'  texts so that javascripts can be loaded correctly, thus svg-dynamic-graphs appear correctly to everyone that is using ReverseProxy sollution or are behind a Nat.

This has been tested on 'svg-dynamic-graphs', and need more tests on other .php files.
I'm not sure, that the 'asset()' function is availlable on all .php files.

But here is my contribution.

Keep up the good work. 

LibreNMS is great !

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Support/FAQ.md [hunk 1]
```diff
@@ -125,6 +125,13 @@ using the [included
 snmpd.conf](https://raw.githubusercontent.com/librenms/librenms/master/snmpd.conf.example)
 file.
 
+If you are behind a Proxy or thru a NAT'd ipaddress with an Ngix, you
```
Comment (murrant) [inline doc/Support/FAQ.md:128]: Typo here Ngix -> Nginx

Comment (Cupidazul) [reply doc/Support/FAQ.md]: thx

Comment (murrant) [reply doc/Support/FAQ.md]: Also, can you be more explicit about what scenarios this will help with?

Comment (Cupidazul) [reply doc/Support/FAQ.md]: Sure, this helps when you have a NAT in front of libre NMS, in my case I'll be doing proxy also, but this is valid for any nginx environment. in apache you have MultiViews , in Nginx you do not. so this is a workaround for that.

If you have an Nginx, and you try to open /device/.../graph.php?args you cannot, and you will see a 404 error.

In this Nginx workaround, what it does is to "try" the "graph.php" in the http_root, where it is in fact. whar way Nginx stripes /device/ part and opens the right "graph.php" in the right path. If not, the 404 was a pointer that nginx was trying to find the "graph.php" inside some path /device/.../.../ that doesnt exists, for that the browser shows a 404 - file not found .

this was my troubleshooting last night, lost some time on this, and this was my fix .

keep up the good work, Librenms is awesome .

Comment (Cupidazul) [reply doc/Support/FAQ.md]: Yup, performance tab is not showing...

SyntaxError: expected expression, got '<'  vis.min.js:1
ReferenceError: vis is not definedtab=performance:682:22

Comment (Cupidazul) [reply doc/Support/FAQ.md]: sorry I might have been mistaken, I'm on a docker image running librenms, and still not sure what version I have...
validade shows 1.63, but i still have some files from previous versions...
ex: ${LIBRENMS_PATH}/includes/html/pages/device/performance.inc.php (should not be there)

Comment (Cupidazul) [reply doc/Support/FAQ.md]: sorry I need to redo this pull request. It's no good this way. sorry

**Pull request closed without merging**